content/docs/getting-started/hello-world.md

```diff
@@ -203,7 +203,7 @@ You can find the full example [here][full-code].
 command starts a listening TCP socket on the previously specified port.
 
 ```bash
-$ nc -l -p 6142
+$ nc -l 6142
```

Comment:
I'm not 100% sure about this, but on Mac OS at least, `-p` causes an error and netcat works without it. I haven't tested on other platforms but this does correspond to some documentation I found.